awx_collection/plugins/modules/tower_workflow_job_template.py

```diff
@@ -142,11 +300,66 @@
     name: example-workflow
     description: created by Ansible Playbook
     organization: Default
+
+- name: Create a workflow job template with schema in template
+  awx.awx.tower_workflow_job_template:
+    name: example-workflow
+    inventory: Demo Inventory
+    extra_vars: {'foo': 'bar', 'another-foo': {'barz': 'bar2'}}
+    schema:
+      - identifier: node101
+        unified_job_template:
+          name: example-project
+          inventory:
+            organization:
+              name: Default
```

Comment:
is the indentation correct here? looks a bit wonky

Reply:
:mag: ok I think I get it....you are trying to indicate the organization of the unified job template indirectly by saying use the JT with name `example-project` that has an inventory (not named) that is in the Default organization.

Is that right?

Reply:
This is for an inventory sync update in a workflow. It's actually the direct output of tower_export and the way it keeps it in the api. The goal is to reduce looping to create nodes of a workflow, and it follows the model from tower_export, which follows how it keeps things in the API. The fact that organization is referenced differently for inventories vs Job templates is just the way current tower api works.